_inc/client/lib/plans/constants.js

```diff
@@ -148,10 +148,9 @@ export function getPlanClass( plan ) {
 			return 'is-business-plan';
 		case PLAN_JETPACK_BACKUP_DAILY:
 		case PLAN_JETPACK_BACKUP_DAILY_MONTHLY:
-			return 'is-daily-backup-plan';
 		case PLAN_JETPACK_BACKUP_REALTIME:
 		case PLAN_JETPACK_BACKUP_REALTIME_MONTHLY:
-			return 'is-realtime-backup-plan';
+			return 'is-jetpack-backup-plan';
```

Comment:
Do we need to update the places where we've started using `is-daily-backup-plan` and `is-realtime-backup-plan` now that we're changing the plan class, to avoid breaking things?

Reply:
This was reverted in favor of the previous functionality.